src/dc_receive_imf.rs

```diff
@@ -1510,7 +1525,7 @@ fn is_reply_to_messenger_message(context: &Context, mime_parser: &MimeMessage) -
     false
 }
 
-fn is_msgrmsg_rfc724_mid_in_list(context: &Context, mid_list: &str) -> bool {
+pub fn is_msgrmsg_rfc724_mid_in_list(context: &Context, mid_list: &str) -> bool {
```

Comment:
isn't `pub(crate)` enough here?

Reply:
Added a commit with this change.

But the whole `dc_receive_imf` can be imported as non-public in `src/lib.rs` actually.